packages/plugins/index.ts

```diff
@@ -1,2 +1,4 @@
 import urlWritebackPlugin from "./urlWriteback/index.ts";
-export { urlWritebackPlugin };
+import createSlugManager from "./slugManager/mod.ts";
```

Comment:
Why is the previous plugin's entrypoint index.ts and this one is mod.ts?

Reply:
I learned that `mod.ts` is the recommended thing: https://deno.land/manual@v1.25.2/contributing/style_guide#do-not-use-the-filename-indextsindexjs. 

I think we should update the other one too, was just going to make that a separate pr.

Reply:
Sounds good